app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+
+class UserController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    public function profile($id){
+        $user = User::where('id',$id)->first();
```

Comment:
PSR coding style violation.
`where('id',$id)`, there should be a space after comma, like this: `where('id', $id)`

Reply:
Fix it in other places too.

Reply:
done